TLDR.md

```diff
@@ -0,0 +1,18 @@
+
+
+# Introduction
+
+Hey! This is Purple Bubble. We're a crew of teens on a mission to make messaging epic again. Zero tracking, spying, or data selling. Everything is end-to-end encrypted, signed, and decentralized. All open source.
```

Comment:
Is this the same copy from the website?

Reply:
yes, but from memory

Reply:
Can we make sure it matches what's on the website for consistency?

Reply:
```suggestion
# CALLING ALL BUBBLERS!
We're Purple Bubble, a crew of teens on a mission to make messaging epic again! Tired of feeling like the product in those other apps? We get it. That's why we're building something new: an app that puts YOU in control. No ads, no data mining, no creepy algorithms. Just you, your friends, and a whole lot of fun.

# READY TO PEEK BEHIND THE CURTAIN?
We're in the super-early stages (think building a pillow fort!), and we're looking for awesome testers to join our inner circle. Sign up below to get the scoop on all things Purple Bubble (promise it'll be way more exciting than your dentist's newsletter - no spam, just 4-5 super cool updates a year).


```

Reply:
this text, but change the newsletter bit to direct people to the website.

Reply:
should I direct people to sign up via the website or just to the website?

Reply:
Direct to sign up via website, traffic there is good too! Our slack should be linked in these docs as well tho